checkout.html

```diff
@@ -1,118 +1,175 @@
+                <!--Cart-->
                 <li>
-                    <a href="#"> <i class="fab fa-youtube"></i></a>
+                    <a href="cart.html"><i class="fas fa-shopping-cart"></i></a>
                 </li>
             </ul>
+        </nav>
+    </header>
+
+    <div class="wrapper">
+        <h1>Shipping information</h1>
+
+        <!--form-->
+        <form class="checkout-form">
+            <hr />
+            <div class="content">
+                <label for="Name">FIRST NAME & LAST NAME</label>
+                <div class="input-control">
+                    <input type="text" id="firstName" placeholder="FIRST NAME .." />
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="input-control">
+                    <input type="text" id="lastName" placeholder="LAST NAME .." />
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+
+                <label for="Street Address">STREET ADDRESS</label>
+                <div class="input-control twoColumns">
+                    <input type="text" id="address" placeholder="STREET ADDRESS .." />
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+
+
+                <label for="City">CITY</label>
+                <div class="input-control">
+                    <select name="city" class="cities" id="cityId">
+                        <option value="">Select City</option>
+                    </select>
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+                <div class="hide"></div>
+
+                <label for="State">STATE</label>
+                <div class="input-control">
+                    <select name="state" class="states" id="stateId">
+                        <option value="">Select State</option>
+                    </select>
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+                <div class="hide"></div>
+
+                <label for="country">COUNTRY</label>
+                <div class="input-control">
+                    <select name="country" class="countries" id="countryId">
+                        <option value="">Select Country</option>
+                    </select>
+                    <i class="fas fa-check-circle"></i>
+                    <i class="fas fa-exclamation-circle"></i>
+                </div>
+                <div class="hide"></div>
+                <div class="hide"></div>
```

Comment:
For the country, state, and city, it feels weird to open the selection box of state and city and not being able to select anything. Maybe try disabling them before the preceding one get selected?

Example use case:
User start off only able to open the selection box of country. State and city are either disabled or hidden.
When user select a country, state becomes enabled, if country is reverted to default than state is disabled again.
When user select a state, city becomes enabled, if anything preceding it cause state to be reverted to default, city is disabled again.

Reply:
created the issue "Order in displaying state, city, country in checkout page #53"